python_modules/dagit/dagit/pipeline_run_storage.py

```diff
@@ -131,7 +132,9 @@ def __init__(self, pipeline_run, start_cursor=None):
 
     def __call__(self, observer):
         self.observer = observer
-        self.observer.on_next(self.pipeline_run.logs_after(self.start_cursor))
+        events = self.pipeline_run.logs_after(self.start_cursor)
```

Comment:
what is going on here

Reply:
Don't send empty list as an event.

Reply:
oh i missed something i thought this was a bigger change